src/pages/CoursesListView/Course.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="course">
+    <router-link
+      :to="`/course/${id}`"
+      class="course-videoWrapper"
+    >
+      <VideoPlayer
+        class="course-video"
+        isPreview
+        :previewImageLink="`${previewImageLink}/cover.webp`"
+        :link="courseVideoPreview?.link"
+      />
```

Comment:
FYI: it can be better to keep one style of adding components: write `video-player` instead of `VideoPlayer`

Reply:
yes, I have problems with it, thank you for the attention

can you help me with naming for it? I use quasar components (q-button), I use custom components (VideoPlayer) and also I use default components like (RouterView)

is it ok to use one naming for different kinds of component?